packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -628,19 +636,19 @@ contract ControllerHelper is UniswapControllerHelper, EulerControllerHelper, IER
                     );
                 } else if (
                     // this will execute if the use case is to mint in vault, deposit collateral in vault or mint + deposit
-                    data[i].rebalanceVaultNftType == ControllerHelperDataType.RebalanceVaultNftType.MintIntoVault
+                    data[i].rebalanceVaultNftType == ControllerHelperDataType.RebalanceVaultNftType.DepositIntoVault
```

Comment:
Dont love the name (new or old), maybe DepositMintVault or something, but not critical.